spec/hand_rankings/find_pair_spec.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../../lib/poker_hands/hand_rankings/find_pair'
```

Comment:
Let's get this gem using `require` instead of `require_relative`

Reply:
Going to do a big sweep right now and do this everywhere.